test/hotspot/jtreg/compiler/intrinsics/TestCompareUnsigned.java

```diff
@@ -30,7 +30,8 @@
  * @test
  * @key randomness
  * @bug 8283726 8287925
- * @requires os.arch=="amd64" | os.arch=="x86_64" | os.arch=="aarch64" | os.arch=="riscv64"
+ * @requires os.arch=="amd64" | os.arch=="x86_64" | os.arch=="aarch64" | os.arch=="riscv64" | os.arch=="ppc64" | os.arch=="ppc64le"
+
```

Comment:
The test expects "CmpU3" for integers to be available. Can you implement that, too, please?